File: packages/insomnia-httpsnippet/test/fixtures/output/python/python3/multipart-data.py
import http.client

conn = http.client.HTTPConnection("mockbin.com")

payload = "-----011000010111000001101001\r\nContent-Disposition: form-data; name=\"foo\"; filename=\"hello.txt\"\r\nContent-Type: text/plain\r\n\r\nHello World\r\n-----011000010111000001101001--\r\n"

headers = { 'content-type': "multipart/form-data; boundary=---011000010111000001101001" }

conn.request("POST", "/har", payload, headers)

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))

File: packages/insomnia-httpsnippet/test/fixtures/output/node/unirest/multipart-data.js
var unirest = require('unirest');

var req = unirest('POST', 'http://mockbin.com/har');

req.headers({
  'content-type': 'multipart/form-data; boundary=---011000010111000001101001',
});

req.multipart([
  {
    body: 'Hello World',
    'content-type': 'text/plain',
  },
]);

req.end(function(res) {
  if (res.error) throw new Error(res.error);

  console.log(res.body);
});
